.github/workflows/publish.yml

```diff
@@ -22,35 +22,18 @@ jobs:
       pyi-ver: ${{ env.PYI_VER }}
       pyi-rel-ver: ${{ env.PYI_REL_VER }}
     steps:
-      - if: github.event_name == 'schedule' || github.event.inputs.workflow-ghidra-ver == 'latest'
-        name: Get Latest Ghidra Version
-        id: get_latest_ghidra_ver
-        uses: pozetroninc/github-action-get-latest-release@v0.6.0
-        with:
-          repository: NationalSecurityAgency/ghidra
-          excludes: prerelease, draft
-      - name: Set Ghidra Version from Latest
-        if: github.event_name == 'schedule' || github.event.inputs.workflow-ghidra-ver == 'latest'
-        id: format_ghidra_ver
-        run: |
-          echo "GHIDRA_VER=$(echo ${{steps.get_latest_ghidra_ver.outputs.release}} | cut -d_ -f2)" >> $GITHUB_ENV
+      - name: Set Ghidra Version to latest
+        run: echo "GHIDRA_VER=latest" >> $GITHUB_ENV
       - name: Set Ghidra Version from Input
         if: github.event_name == 'workflow_dispatch' && github.event.inputs.workflow-ghidra-ver != 'latest'
-        run: |             
-          echo "GHIDRA_VER=$(echo ${{github.event.inputs.workflow-ghidra-ver}})" >> $GITHUB_ENV
-      - name: Checkout repo
-        uses: actions/checkout@v3
+        run: echo "GHIDRA_VER=$(echo ${{github.event.inputs.workflow-ghidra-ver}})" >> $GITHUB_ENV
       - name: Get Ghidra Stubs Version
-        id: get_pyi_ver
-        run: |
-          echo "PYI_VER=$(python version.py)" >> $GITHUB_ENV
+        run: echo "PYI_VER=$(python version.py)" >> $GITHUB_ENV
       - name: Get Ghidra Stubs Full Release Version
-        id: get_pyi_rel_ver
-        run: |
-          echo "PYI_REL_VER=$GHIDRA_VER.$PYI_VER" >> $GITHUB_ENV
+        run: echo "PYI_REL_VER=$GHIDRA_VER.$PYI_VER" >> $GITHUB_ENV
```

Comment:
If `GHIDRA_VER` is always set to `latest` before, then `PYI_REL_VER` would always be `latest.X.Y.Z` instead of containing the actual Ghidra version number. I'm not sure where that variable is actually _used_ but the packages published to PyPI definitely should contain the actual Ghidra version number and not `latest`